### PR TITLE
document using ESLint in CRNA project

### PR DESCRIPTION
**Pull request by peterlazar1993**
- add steps to install eslint
- provide a sample configuration

### CODE REVIEW

**Comment by peterlazar1993** [comment]
@dikaiosune This PR should take care of #185.

cc: @marcdel

File: react-native-scripts/template/README.md [hunk 1]
```diff
@@ -142,6 +143,47 @@ You can optionally use a [plugin for your IDE or editor](https://flow.org/en/doc
 
 To learn more about Flow, check out [its documentation](https://flow.org/).
 
+## Adding ESLint
+
+[ESLint](http://eslint.org/) is a configurable javascript linter. ESLint can help standardize a style guide across the team.
```
Comment (brentvatne) [inline react-native-scripts/template/README.md]: javascript => JavaScript

File: react-native-scripts/template/README.md [hunk 2]
```diff
@@ -142,6 +143,47 @@ You can optionally use a [plugin for your IDE or editor](https://flow.org/en/doc
...
+
+To add eslint to a Create React Native App project, follow these steps:
+
+1. Run `npm install --save-dev eslint eslint-plugin-react`
```
Comment (brentvatne) [inline react-native-scripts/template/README.md]: people should probably use react-native preset, or eslint-config-react-native-prettier

File: react-native-scripts/template/README.md [hunk 3]
```diff
@@ -142,6 +143,47 @@ You can optionally use a [plugin for your IDE or editor](https://flow.org/en/doc
...
+2. Add `"lint": "eslint ."` to the `scripts` section of your `package.json`.
+3. Create a `.eslintrc` file in the project root which is used to configure how eslint should run.
+
+After configuring eslint, run `npm run lint` (or `yarn lint`) to check for linting erros.
+
+Here's a minimal eslint configuration file:
```
Comment (brentvatne) [inline react-native-scripts/template/README.md]: minimal 🤔

**Comment by brentvatne** [comment]
hey thanks for writing this! I think we need to tailor it more towards react-native, in particular https://www.npmjs.com/package/eslint-config-react-native and mention prettier, which is very valuable with eslint imo

**Comment by peterlazar1993** [comment]
@brentvatne 
I'm not used to branch based development workflow. I guess rebasing the branch with upstream was a bad idea?
Let me know if I should submit a fresh pull request.

Also I've updated the docs.

**Comment by brentvatne** [comment]
@peterlazar1993 - rebasing with upstream is usually a good call but this commit history does look pretty weird. might be worth squashing the commits into one and re-opening a new PR :)

**Comment by MrLoh** [comment]
why has this been closed, it seemed like a helpful addition, I used the explanation in the readme to setup my eslint

**Comment by peterlazar1993** [comment]
@MrLoh This is being worked on in this pull request https://github.com/react-community/create-react-native-app/pull/338

**Pull request closed without merging**